scripts/build.js

```diff
@@ -32,16 +32,26 @@ webpack(config).run(function(err, stats) {
   var openCommand = process.platform === 'win32' ? 'start' : 'open';
   var homepagePath = require(path.resolve(relative, 'package.json')).homepage;
```

Comment:
I think we have a path issue here - it is looking for package.json in my user directory. Maybe something like `path.resolve(__dirname, relative, '..', 'package.json')`?